tool/lib/commands/build.dart

```diff
@@ -95,8 +95,6 @@ class BuildCommand extends Command {
             '--web-renderer',
             'canvaskit',
             '--pwa-strategy=offline-first',
-            // Enable default optimizations: https://dart.dev/tools/dart-compile#js
-            '--dart2js-optimization=O1',
```

Comment:
do we need the `--release` arg here?

Reply:
Nope, that's passed as the `buildMode` (below)